internal/states/statemgr/testing.go

```diff
@@ -155,11 +155,12 @@ func TestFullInitialState() *states.State {
 		Type: "null_resource",
 		Name: "foo",
 	}
+	/* TODO this does not actually create a resource, instances are required
```

Comment:
Do we just drop this TODO?

Reply:
Done in [83c8ef9](https://github.com/opentofu/opentofu/pull/2065/commits/83c8ef97a4247ac1e0e12436b9a70e2295bbd505)